pyproject.toml

```diff
@@ -59,7 +59,8 @@ dependencies = [
   "cachetools",
   "python-multipart",  # see https://www.starlette.io/#dependencies
   "arize-phoenix-evals>=0.13.1",
-  "pyyaml", # for OpenAPI
+  "fastapi",
+  "pydantic>=1.0,<3",
```

Comment:
I still don't understand why we want this as explicit deps @axiomofjoy - don't we want it as transitive and align with what fastapi deems right?